src/applications/vaos/components/AppointmentTime.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { formatInTimeZone } from 'date-fns-tz';
+import { stripDST } from '../utils/timezone';
+
+/**
+ * Component for displaying appointment time in a consistent format with timezone
+ */
+export default function AppointmentTime({ date, timezone }) {
+  const formattedTime = stripDST(
```

Comment:
nit: FYI stripDST is only used for timezone abbreviations (e.g. EST, EDT, ...) so there's no need to apply it to formattedTime.